### DISCUSSION TITLE
Only one field isn't updating and response timeout (over 10.02 sec)

### DISCUSSION THREAD

**Discussion by mimranzulkfal** [supabase/supabase#10383] [Questions]
# Bug report

## Describe the bug

Actually, I have to create or update reminder schedule time but issue is there in one field which is time (reminder time isn't updating in db). 

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Click on edit icon 
![image](https://user-images.githubusercontent.com/99423116/201681534-be41b3b3-14f1-45ad-a197-45609ebbfb37.png)
2. When we update time, click save but can't getting data back (timeout error)
![image](https://user-images.githubusercontent.com/99423116/201681859-bef6f6e9-6174-4862-b0d0-6e450a0b5d08.png)
5. See error
![image](https://user-images.githubusercontent.com/99423116/201682035-05b03182-f91e-40bc-b9ac-f8f0b6f36b4f.png)

## Expected behavior

So, I'm adding screenshots for code flow: 
- Using graphql functions to create and update
![image](https://user-images.githubusercontent.com/99423116/201682469-772b2dc3-6a8e-474c-ba6d-a4dc4fc4598a.png)
- These are the fields we're storing:
![image](https://user-images.githubusercontent.com/99423116/201682775-ffd6b3fa-b5dc-4927-ad32-eb47b6d3f4a6.png)
- And here is the url to fetch but on time of creation and updation, we're getting data as undefined: 
![image](https://user-images.githubusercontent.com/99423116/201683033-a8f4508a-64ae-461a-a8e9-6728fb72d5da.png)

## Screenshots

Attached already

## System information

- OS: macOS
- Browser (if applies) Chrome
- Version of supabase-js: 1.25.1
- Version of Node.js: v16.17.0

**Comment by awalias** [comment]
I'll convert this one into a discussion as it will likely get more eyeballs. You might also consider dropping your schema in here and also the endpoint you're posting to etc., I don't think it's possible to help on this one without some more details.
